extension.json

```diff
@@ -1,30 +1,24 @@
 {
   "name": {
-    "en": "Name of extension (required in at least one language)"
+    "en": "Document Publisher"
   },
   "description": {
-    "en": "A brief description of what this extension does, this will be pulled into the main docs as a link (required in at least one language)"
+    "en": "Adds a publisher field to the document object."
   },
   "documentationUrl": {
-    "en": "http://httpbin.org/anything/path/to/documentation/in/specified/language"
+    "en": "https://github.com/open-contracting-extensions/ocds_document_publisher_extension"
   },
   "schemas": [
-    "record-package-schema.json",
-    "release-package-schema.json",
     "release-schema.json"
   ],
-  "codelists": [
-    "emptyCodelist.csv"
-  ],
   "compatibility": [
     "1.1"
   ],
-  "dependencies": [
-    "http://httpbin.org/anything/path/to/extension1/extension.json",
-    "http://httpbin.org/anything/path/to/extension2/extension.json"
+  "testDependencies": [
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_eu_extension/1.1/extension.json"
```

Comment:
I'm unsure why you've included this? Is it just so the example in the README passes? If so see my previous comment about that.

Reply:
Yep, that was it. I'll wait for that discussion thread to be resolved and then update if needed :-)

Reply:
Removed in https://github.com/open-contracting-extensions/ocds_document_publisher_extension/pull/1/commits/26e2570c72622962e674e422a40badc4c199cb46